### DISCUSSION TITLE
Does anyone have [env:tasmota32c3-mi32-homebridge] working?

### DISCUSSION THREAD

**Discussion by kskenyon** [arendst/Tasmota#17722] [Q&A]
Does anyone have [env:tasmota32c3-mi32-homebridge] working?  I successfully compiled and uploaded it to an esp32c3stamp.  However, it does not respond to any of the MI32 commands.  All of them just say unknown in the console. The web interface is just the standard one, too.  Is the M5Stamp supported for this code and does anyone have any pointers how to get it to work if it is?  It boots OK and seems to run OK in the serial monitor without responding to any MI32 commands.  I have studied the wiki https://tasmota.github.io/docs/Bluetooth_MI32/#commands.  I'd like to use it to read a miflora through bluetooth.

**Comment by Jason2866** [comment]
Have you enabled BLE with the command `SetOption105 1`?
You find an precompiled under section unofficial https://tasmota.github.io/install/ here

**Comment by kskenyon** [comment]
No luck.  MI32Option1 0 or any other command just gives command unknown.  Must be missing something. I uploaded the precompiled version you linked to.  

SO105 1 is not in the Tasmota command wiki.  It did accept that command.
BLE-Log:

15:56:35: Searching for devices
15:56:39: Connecting to: Flower care
15:56:44: Received advertisement from "Flower care"...
15:57:06: Detected Mi Device
15:57:15: Disconnected.


No data.  I’m on a MacBook Pro M1.  Maybe that has something to do with it?

**Comment by Jason2866** [comment]
BLE is working. Device found, not working is a different thing. 
What do you mean with no data on the PC???

Are you trying to connect to decode the key?
https://tasmota.github.io/docs/Bluetooth_MI32/#commands

It does work the author has a mac pro m1 too. You have to use Chrome Browser not the crippled Safari

**Comment by Jason2866** [comment]
And sorry the command is `SetOption115 1` to enable BLE. Should not mention commands out of mind...

**Reply by kskenyon** [reply]
MiFlora is working unencrypted.   Can't add the bridge to HomeKit right now.  Possibly Apple problem.
Could add SO115 1 to the Homebridge Wiki page in case someone else lands there without reading the BLE pages first.

Thanks for your pointer.

**Comment by Jason2866** [comment]
Doku is Open Source too. Just click the pencil in upper right corner.
Mac M1 does work, i get the key for the BLE thermometer.

**Reply by kskenyon** [reply]
Good to know.  The only Mi device I have is MiFlora.  It doesn't seem to have a key.  FW 3.3.5
